### PR TITLE
Support Python 3.7

### PR DESCRIPTION
**Pull request by dahlia**

### CODE REVIEW

**Comment by kanghyojun** [comment]
The test was failed on both [Python 3.4] and [Python 3.7]. I think a running rebuild is worked on [Python 3.4] although [Python 3.7] problem is not.

[Python 3.7]: https://travis-ci.org/spoqa/settei/jobs/415345442
[Python 3.4]: https://travis-ci.org/spoqa/settei/jobs/415345445

**Comment by kanghyojun** [comment]
@dahlia The reason why the test was failed is Travis-CI dosen't support Python 3.7 yet. Consider bypassing it like I did in [this file](https://github.com/admire93/pyleftpad/blob/master/.travis.yml#L7-L12).

**Comment by dahlia** [comment]
@admire93 @hanc1208 Fixed the Python 3.7 build which was failed.  Python 3.4 build is still failing due to the changelog check and IMHO it's okay to ignore.

File: settei/base.py [hunk 1]
```diff
@@ -36,11 +36,17 @@ def get_union_types(type_) -> bool:
...
         if isinstance(type_, typing.UnionMeta):
             return type_.__union_params__
-else:
+elif hasattr(typing, '_Union'):
     # For older versions of typing
     def get_union_types(type_) -> bool:
         if type(type_) is typing._Union:
             return type_.__args__
+else:
+    # For newer versioss of typing (>= Python 3.7)
```
Comment (hanc1208) [inline settei/base.py]: typo `versioss` -> `versions`

**Review by hanc1208 — CHANGES_REQUESTED** [review]

**Comment by kanghyojun** [comment]
I think we can mere this PR after fixing one problem @hanc1208 mentioned.

File: settei/base.py (hunk 1 above)
Comment (dahlia) [reply settei/base.py]: Fixed: cc47a82b46848d858fe6620b5371fc7415f96f35

**Review by kanghyojun — APPROVED** [review]

**Review by hanc1208 — APPROVED** [review]

**Pull request merged**